.github/workflows/ci.yml

```diff
@@ -51,3 +51,19 @@ jobs:
                   npm test
               env:
                   CI: true
+
+    jsr_test:
+        name: Verify JSR Publish
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: "lts/*"
+
+            - name: Install Packages
+              run: npm install
+
+            - name: Run --dry-run
+              run: npm run test:jsr
```

Comment:
It needs to run build. It's fine to add it to the above job - after `npm test`. :)

Reply:
Good catch!